### DISCUSSION TITLE
Hidden IP Client?

### DISCUSSION THREAD

**Discussion by alkinabc** [rustdesk/rustdesk#8982] [Q&A]
Hello,
I would use RustDesk in the company I work for. For this to work, the server IP must not be visible. Is there a way no one could see the server IP? maybe with LDAP?
I would be very happy to receive an answer.
Best regards

**Comment by basilgello** [comment]
Put the server in some meshnet like Yggdrasil using https://github.com/yggdrasil-network/yggstack (full disclosure: I am maintaining it)

But it sounds like an XY problem to me. What is real reason the server must not be known? A yone with tcpdump or netstat can see the connection list.

**Reply by alkinabc** [reply]
Firstly, thank you for your answer. I'll definitely take a look at it right away. 
The reason why I don't want people to see the server credentials is that I don't want everyone to be able to use our server to connect to RustDesk. And yes, you can find out the IP but not the key, which is why it would be good if I had the option to hide the server credentials.

**Reply by basilgello** [reply]
So the problem is not with server IP but with some sort of centralized authentication needed. @rustdesk LDAP is there in Pro server right?

**Reply by alkinabc** [reply]
Yes, with the RustDesk Pro version you would have the option of using an LDAP connection. Do you happen to know if it is the way I imagine it to be when I used ldap, i.e. that everyone who uses it has to log in first and only then is it decided which server credentials they will be logged in with?
